core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -263,6 +269,12 @@ angular.module('oppia').component('contributionsAndReview', {
       };
 
       ctrl.onClickViewSuggestion = function(suggestionId) {
+        if (submissionIsInProgress) {
+          AlertsService.addInfoMessage(
+            'Previous suggestion is being submitted. ' +
```

Comment:
"The previous suggestion is still being submitted. Please ..."

(But why are such suggestions even viewable in the first place? Shouldn't we make them non-viewable but show something like "Submission in progress..."?)

Reply:
Is it possible to remove the suggestion from the review list after submitting the review but before exiting from the modal?

Reply:
I've changed this such that, I remove the selected suggestion immediately from the list on selecting accept/reject in the review modal.

Reply:
Thanks, sounds good!